draft-ietf-idr-link-bandwidth.xml

```diff
@@ -158,25 +162,54 @@
         <t>An originator of the link bandwidth community SHOULD be able to
         originate either a transitive or a non-transitive link bandwidth
         extended community. Implementation SHOULD provide configuration to set
-        the transitivity type of the link bandwidth community.&nbsp;No more
+        the transitivity type of the link bandwidth community, as well as Global
+        Administrator filed value and bandwidth value in (Local Administrator
+        filed), trough local policy.&nbsp;No more
         than one link bandwidth extended community SHALL be attached to a
         route.</t>
+        <t>An originator can attach link bandwidth community to BGP path in
+        egress processing to adj-RIB-out only or in ingress processing in which
+        case link bandwidth community is present in Local-RIB.</t>
+        <section anchor="Egress"
```

Comment:
I understand the intent of adding this egress/ingress differentiation. 

But during the egress processing, I fear implementations may not be able to differentiate between:
  - whether LBW was received in rib-in on wire
  - whether LBW was originated in rib-in by local node
  - whether LBW was originated in rib-out by local-node
It may be hard to observe compliance too, because it is hard to tell what each implementation does, from looking at external behavior. :)

This text is a good segway to the rfc4360-bis discussion we were having, that this behavior needs to be clarified for all extendended-communities in general, not only LBW.

Since this discussion was kept for rfc4360-bis and is also clarified by draft-bess-dmz, we had not put in this draft. But may be we can put some words.. we can discuss what others feel. 

I like the following text you added:

     <t>Note: Implementation may provide configuration option (knob) to allow sending
          of non-transitive link bandwidth communities to external BGP sessiosns.</t>

May be that takes care of everything specific to LBW usecases. (make it a normative MAY)